utilities/mongodb.js

```
@@ -1,196 +1,200 @@
+		new Promise(async (resolve, reject) => {
+			try {
+				const client = new MongoClient(mongoDBUrl);
+				await client.connect();
+				console.log('✅ Connected to MongoDB');
+				resolve(client);
+			} catch (error) {
+				console.error('❌ MongoDB connection failed:', error);
+				reject(error);
+			}
+		}),
+	// getCollections: async (collectionName) =>
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)